Profile Folder/chrome/locale/hr-HR/dtd/gsettings.dtd

```diff
@@ -147,11 +147,11 @@
 
 <!ENTITY gsettings.peoplePositionTitle			"Pozicija">
 <!ENTITY gsettings.peoplePositionDescription	"Gumb za ljude imao je različite pozicije tijekom godina.">
-<!ENTITY gsettings.tabsToolbar                  "[!!! Táβƨ Tôôℓβář !!!]">
-<!ENTITY gsettings.titlebarButtons              "[!!! Tïƭℓèβář βúƭƭôñƨ !!!]">
+<!ENTITY gsettings.tabsToolbar                  "Alatna traka kartica">
+<!ENTITY gsettings.titlebarButtons              "Gumbi naslovne trake">
 
-<!ENTITY gsettings.oldButton		"[!!! Óℓδ βúƭƭôñ !!!]">

```

Comment:
I am so sorry for this but last night I didn't verify the strings properly because I had to go to bed and ended up sending out an outdated one :(
This commit updates it:
https://github.com/angelbruni/Geckium/commit/66cd45532db9c148debc7f9d13333f5c258dd0bd
Once again, I am sorry.